projects/angular-auth-oidc-client/src/lib/callback/interval.service.ts

```diff
@@ -1,6 +1,6 @@
-import { inject, Injectable, NgZone } from '@angular/core';
+import { inject, Injectable, NgZone, DOCUMENT } from '@angular/core';
```

Comment:
These changes have caused the build to fail for earlier versions of Angular.
Is this change required or can we keep it as is?

If we can't keep it, this is a breaking change and:
- needs to be documented
- the build pipeline needs to be updated